### PR TITLE
Integration test with embedded mongodb 

### PR DESCRIPTION
**Pull request by dapariciom**
Integration test with embedded mongodb

### CODE REVIEW

File: src/test/java/com/example/shorturl/integration/test/MongoDbTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    private UrlRepository urlRepository;
+
+    @AfterEach
+    void cleanUp() {
+        this.urlRepository.deleteAll();
+    }
+
+    @Test
+    void saveAndReturnUrlsFromRepository() {
+        this.urlRepository.save(UrlEntity.builder().id(1).originalUrl("https://www.youtube.com/").build());
```
Comment (BeeAM17) [inline src/test/java/com/example/shorturl/integration/test/MongoDbTest.java]: Do we need `this` here?

File: src/test/java/com/example/shorturl/integration/test/MongoDbTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        this.urlRepository.save(UrlEntity.builder().id(2).originalUrl("https://www.facebook.com/").build());
+        this.urlRepository.save(UrlEntity.builder().id(3).originalUrl("https://www.instagram.com/").build());
+        List<UrlEntity> urlList = urlRepository.findAll();
+        assertEquals(3, urlList.size());
```
Comment (BeeAM17) [inline src/test/java/com/example/shorturl/integration/test/MongoDbTest.java]: We need to assert if our URLs got shortened too

**Review by BeeAM17 — COMMENTED** [review]
Small comments

File: src/test/java/com/example/shorturl/integration/test/MongoDbTest.java (hunk 1 above)
Comment (dapariciom) [reply src/test/java/com/example/shorturl/integration/test/MongoDbTest.java]: Fixed

File: src/test/java/com/example/shorturl/integration/test/MongoDbTest.java (hunk 2 above)
Comment (dapariciom) [reply src/test/java/com/example/shorturl/integration/test/MongoDbTest.java]: MongoDbTest is just testing if we are able to read and write directly to the repository. In this test we save the entities directly to the urlRepository and check if they are on the database. We are not calling the complete implementation, that is checked on UrlIntegrationTestV1

**Review by BeeAM17 — APPROVED** [review]
Good job!

**Pull request merged**